src/browser/builds/algoliasearch.angular.js

```diff
@@ -34,7 +34,11 @@ window.angular.module('algoliasearch', [])
     }
 
     algoliasearch.version = require('../../version.js');
-    algoliasearch.ua = 'Algolia for AngularJS (' + algoliasearch.version + ')';
+
+    algoliasearch.ua =
+      'Algolia for JavaScript (' + algoliasearch.version + '); ' +
+      'AngularJS (' + window.angular.version.full + ')';
```

Comment:
can nothing of this be undefined?

Reply:
`window.angular` is defined for sure otherwise the script has already thrown [a couple of lines above](https://github.com/algolia/algoliasearch-client-javascript/blob/5de2ccd84320ea967eb02b543d8501fdbb7a3a64/src/browser/builds/algoliasearch.angular.js#L24). The `version` is available on the `angular` object since [v0.9.18](https://github.com/angular/angular.js/blob/master/CHANGELOG.md#0918-jiggling-armfat-2011-07-29).